README.md

```diff
@@ -57,25 +59,15 @@ By default, ESLint will attempt to fix whatever errors possible; the rest should
 #### Links
 Resource | URL
 ------------ | -------------
- API | https://api.hackpsu.org/v1
- API Documentation | https://api.hackpsu.org/v1/doc
+ API | https://api.hackpsu.org/
+ API Documentation | https://api.hackpsu.org/v2/doc
  API Documentation framework | https://apidocjs.com
  MochaJS | https://mochajs.org
  ChaiJS | https://chaijs.com
  CI | https://travis-ci.com
- 
+
  #### ACL details
  All routes under ```/admin``` require some form of ACL permissions. These permissions are configured
  under the authentication from Firebase. To add permissions to your login, contact an administrator.
- The ACL levels are as follows:
- 
- Privilege Level | Definition
- ------------ | -------------
- 1 | Volunteer
- 2 | Team Member
- 3 | Exec member
- 4 | Tech admin
-
-
- 
-  
+
+
```

Comment:
Is there a reason why these were removed?

Reply:
I felt they were not relevant to the frontend, do you disagree?